content/docs/use-cases/sharing-resources-efficiently.md

```diff
@@ -0,0 +1,96 @@
+# Sharing Resources Efficiently
+
+Data science teams need to handle large files, rotate the use of special
+processors, and minimize data transfers. This involves provisioning and managing
+resources such as massive on-prem or cloud data stores and powerful servers,
+which is expensive and time consuming.
+
+![](/img/resource-pool.png) _Fragile and disjointed pool of limited resources_
+
+Minimizing operational cost and complexity can make the difference, for example,
+when:
+
+- Multiple users work on the same shared server.
+- There's a centralized data storage unit or single-entry cluster.
```

Comment:
`or single-entry cluster.` - no sure I understand this term

Reply:
are all these statement _or_ or _and_

Reply:
Or. It's a list of example situations where "Minimizing cost and complexity can make the difference".